src/main/scala/au/com/agiledigital/toolform/model/Resource.scala

```diff
@@ -21,10 +21,11 @@ import com.typesafe.config.Config
 final case class Resource(id: String,
                           resourceType: String,
                           settings: Option[Config],
-                          image: String,
+                          image: String = "NoImageName",
```

Comment:
I'd rather this be treated as an Option[String] to avoid unexpected behaviour

Reply:
Doing that has some weird errors throughout the code. I havent been able to fix them all and would like your help to figure it out.
I've left this as is for now until I can figure out how to successfully make it an Option[String]

Reply:
Done, it is now an Option[String]